jetty-home/src/main/resources/modules/logging-log4j1.mod

```diff
@@ -0,0 +1,59 @@
+# DO NOT EDIT - See: https://www.eclipse.org/jetty/documentation/current/startup-modules.html
+
+[description]
+Configure jetty logging to use Log4j Logging
+SLF4J is used as the core logging mechanism.
+
+[tags]
+logging
+log4j
+
+[depends]
+resources
+
+[provides]
+logging
+log4j
+
+[files]
+basehome:modules/logging/log4j1
+maven://org.slf4j/slf4j-log4j12/${slf4j.version}|lib/logging/slf4j-log4j12-${slf4j.version}.jar
+maven://log4j/log4j/${log4j1.version}|lib/logging/log4j-${log4j.version}.jar
+
+[lib]
+lib/logging/slf4j-api-${slf4j.version}.jar
+lib/logging/slf4j-log4j12-${slf4j.version}.jar
+lib/logging/log4j-${log4j.version}.jar
+
+[ini]
+slf4j.version?=2.0.0-alpha1
```

Comment:
Why does every module have to specify the slf4j.version? can't that be done once in a common module that also provides the library and license?